go.mod

```
@@ -1,8 +1,8 @@
 module github.com/rook/rook
 
-go 1.23.0
+go 1.24.1
```

```suggestion
go 1.24.5
```

Why not latest version?

---

@subhamkrai and which toolchain version?